cookbooks/cdo-nginx/metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'All rights reserved'
 description      'Installs/Configures cdo-nginx'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.0.3'
+version          '0.0.5'
```

Comment:
I was using v `0.0.4` of this cookbook for development-testing purposes, so i thought I'd just bump this one version above that.